File: pangram/Pangram.swift
import Foundation

class Pangram {
    func checkIfPangram(_ sentence: String) -> Bool {
        var set: Set<Character> = []
        for character in sentence {
            if !set.contains(character), character.isLetter {
                set.insert(character)
            }
        }
        return set.count == 26
    }
}

File: pangram/Pangram.cs
using System.Collections.Generic;

public class Pangram {
    public bool CheckIfPangram(String sentence) {
        HashSet<Character> set = new HashSet<>();
        for(int i = 0; i < sentence.Length; i++) {
            char character = sentence[i];
            if(!set.Contains(character) && Character.IsLetter(character)){
                set.Add(character);
            } 
        }
        return set.Count == 26;
    }
}
